feature_engine/outliers/base_outlier.py

```diff
@@ -149,23 +149,29 @@ def __init__(
         self,
         capping_method: str = "gaussian",
         tail: str = "right",
-        fold: Union[int, float] = 3,
+        fold: Union[int, float, Literal["auto"]] = "auto",
         variables: Union[None, int, str, List[Union[str, int]]] = None,
         missing_values: str = "raise",
     ) -> None:
 
         if capping_method not in ["gaussian", "iqr", "quantiles", "mad"]:
             raise ValueError(
-                "capping_method takes only values 'gaussian', 'iqr', 'mad', 'quantiles'"
+                "capping_method takes only values 'gaussian', 'iqr', 'mad', 'quantiles'."
             )
 
         if tail not in ["right", "left", "both"]:
-            raise ValueError("tail takes only values 'right', 'left' or 'both'")
-
-        if fold <= 0:
-            raise ValueError("fold takes only positive numbers")
-
-        if capping_method == "quantiles" and fold > 0.2 and fold != 3:
+            raise ValueError("tail takes only values 'right', 'left' or 'both'.")
```

Comment:
```suggestion
            raise ValueError(f"Parameter ``tail`` takes only values 'right', 'left' or 'both', but got {tail}")
```
Or perhaps use also a Literal for the parameter `tail`.